example/App.vue

```diff
@@ -40,6 +46,12 @@ export default class App extends Vue {
   // method
   greet () {
     alert('greeting: ' + this.msg)
+    this.$refs.helloComponent.sayHello()
```

Comment:
I think this causes compile error since all `.vue` files declared as `Vue`.
Did this example work in your machine?

Reply:
@ktsn  helloComponent is decalred as Hello (see 54 line)

Reply:
Oh, I thought `Hello` should be also typed as `typeof Vue` because we has [`sfc.d.ts`](https://github.com/vuejs/vue-class-component/blob/master/example/sfc.d.ts). But actually webpack resolves actual type. I didn't know that 😄